lib/android.dart

```diff
@@ -363,8 +372,8 @@ Future _getMainActivityKotlinPath() async {
 bool _needToUpdateMainActivity(String language, List<String> lines) {
   bool foundExisting = false;
 
-  String javaLine = 'boolean flutter_native_splash = true;';
-  String kotlinLine = 'val flutter_native_splash = true';
+  String javaLine = 'boolean flutterNativeSplash = true;';
+  String kotlinLine = 'val flutterNativeSplash = true';
```

Comment:
My bad on using underscore instead of camelCase in the first place. But changing this now will duplicate code and produce unexpected behavior on existing projects.

If we choose to change the variable to camelCase, we still need to check for `flutter_native_splash` for backwards compatibility in this method.